.travis.yml

```diff
@@ -1,4 +1,8 @@
 language: node_js
 
 node_js:
-  - 0.10
+  - "4.0"
```

Comment:
Makes sense, it *shouldn't* have to be a `string` to work with `nvm`...
I was just pointing out that travis don't *appear* to be supporting version **5** of node yet. :+1: